railties/test/generators/app_generator_test.rb

```diff
@@ -437,6 +437,12 @@ def test_generator_if_skip_action_cable_is_given
     end
   end
 
+  def test_generator_if_skip_git_is_given
+    run_generator [destination_root, "--skip-git"]
+
```

Comment:
I know, it would do exactly what you did with a new line but not in case of a two line test.